corehq/apps/toggle_ui/static/toggle_ui/js/edit-flag.js

```diff
@@ -32,21 +32,22 @@ hqDefine('toggle_ui/js/edit-flag', [
         };
 
         self.init_items = function (config) {
-            var items = config.items,
-                lastUsed = config.last_used || {},
-                serviceType = config.service_type || {};
-            self.items.removeAll();
-            _.each(_.sortBy(items), function (item) {
-                var fields = item.split(':'),
-                    namespace = fields.length > 1 ? fields[0] : 'user',
-                    value = fields.length > 1 ? fields[1] : fields[0];
-                self.items.push({
-                    namespace: ko.observable(self.padded_ns[namespace]),
-                    value: ko.observable(value),
-                    last_used: ko.observable(lastUsed[value]),
-                    service_type: ko.observable(serviceType[value]),
+            const lastUsed = config.last_used || {},
+                serviceType = config.service_type || {},
+                items = _.map(config.items, function (item) {
+                    var fields = item.split(':'),
```

Comment:
ES6 half the way ;)

Reply:
I am terrible with that. I'll get there one day.